releases/cloud/point-2024-05-09-release-note.md

```diff
@@ -0,0 +1,56 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.3.48.11
+---
+
+# May 09, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## New Features
+
+* **New policy available:** Private Workspaces Shared with Everyone
+
+  * **Private Workspaces Shared with Everyone** are private workspaces that have given access to a large number of users or groups. 
```

Comment:
are private workspaces shared with large number of users through Everyone/Everyone exc. ext. user/All Users